### ISSUE TITLE
Wrtie E2E tests for Packet Forward Middleware

### ISSUE THREAD

**Issue by ninabarbakadze** [celestiaorg/celestia-app#3241]
## Description

We're encountering some issues with the integration of Interchaintest. More context can be found in this related issue regarding [testing ICA](https://github.com/celestiaorg/celestia-app/issues/3207)

- [ ] Write E2E tests for PFM after the team figures out a consistent way of testing all our IBC-based frameworks.

**Comment by rootulp** [comment]
Blocked on interchaintest.

**Comment by ninabarbakadze** [comment]
interchaintest is flakey and hard to work with. I'm trying to get help from the strangelove team.

**Comment by rootulp** [comment]
Closing this as won't do. Instead we're pursuing a unit test for PFM.
